### PR TITLE
Login/registration

### PR DESCRIPTION
**Pull request by LutherNa**

### CODE REVIEW

**Review by AaronLitton — APPROVED** [review]
Looks good

**Review by Justin-Sherfey — APPROVED** [review]

File: src/test/java/com/revature/RevRelay/services/UserServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.test.util.ReflectionTestUtils;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.Assert.assertTrue;
```
Comment (JennicaLeClerc) [inline src/test/java/com/revature/RevRelay/services/UserServiceTest.java:20]: This should probably be Assertions.assertTrue....

**Review by JennicaLeClerc — COMMENTED** [review]
Just need to fix that one line I think so it points to the correct assertions in testing

**Review by JennicaLeClerc — APPROVED** [review]
Looks good

**Pull request merged**